packages/data-api/src/__tests__/testFetchDataGroup.test.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+import { getTestDatabase } from '@tupaia/database';
+
+import { TupaiaDataApi } from '../TupaiaDataApi';
+
+describe('fetchDataGroup()', () => {
+  const api = new TupaiaDataApi(getTestDatabase());
+
+  it('returns a data group in the correct format', async () => {
+    await expect(api.fetchDataGroup('BCDTEST')).resolves.toStrictEqual({
+      code: 'BCDTEST',
+      name: 'Basic Clinic Data Test',
+    });
+  });
+
+  it('returns a data group with data elements in the correct format', async () => {
+    await expect(
+      api.fetchDataGroup('BCDTEST', ['BCD1TEST', 'BCD57TEST'], {
+        includeOptions: false,
+      }),
+    ).resolves.toStrictEqual(
+      expect.objectContaining({
```

Comment:
Only just learnt you can nest jest matchers like this, it's super handy! 🎉

Reply:
Oh, very cool!